### PR TITLE
Feature/display screenshot 2

### PR DESCRIPTION
**Pull request by RV-Adrian**
@ghislaineguerin 
Please validate, and see my notes inline to the changes. Thanks!

### CODE REVIEW

File: src/core/layouts.scss [hunk 1]
```diff
@@ -2,7 +2,22 @@
    ========================================================================== */
 
 .l_display-screenshot {
-	.panel {
+	
+	.panel-body {
```
Comment (RV-Adrian) [inline src/core/layouts.scss]: I had to change from .panel to .panel-body to get the screenshot images to play nicely.

File: src/sections/displays.scss [hunk 2]
```diff
@@ -1,55 +1,4 @@
...
-//   }
-// }
-
-
-/* Display Status Circle
-   ========================================================================== */
-
-
-/* Deprecate
+/* Deprecate Below
```
Comment (RV-Adrian) [inline src/sections/displays.scss]: I've left the class names below this comment. I can try and integrate them as part of the CSS cleanup, and remove this SCSS file.

File: src/core/layouts.scss (hunk 1 above)
Comment (ghislaineguerin) [inline src/core/layouts.scss]: That's ok. Thanks

File: src/sections/displays.scss (hunk 2 above)
Comment (ghislaineguerin) [inline src/sections/displays.scss]: Awesome. Nice to see code being removed :)

**Comment by ghislaineguerin** [comment]
:thumbsup:

**Pull request merged**